src/Resources/contao/templates/backend/be_pagination.html5

```diff
@@ -1,8 +1,9 @@
 
-<div class="pagination <?= $this->class ?>">
+<nav class="pagination <?= $this->class ?>" role="navigation" aria-label="<?= $GLOBALS['TL_LANG']['MSC']['paginationAriaLabel'] ?>">
+  <a href="<?= $this->request ?>#skipNavigation" class="invisible"><?= \StringUtil::specialchars($GLOBALS['TL_LANG']['MSC']['skipNavigation']); ?></a>
 
   <p><?= $this->total ?></p>
-
+    
```

Comment:
@leofeyer whats exactly? The empty row, the blanks or the complete template change?